### ISSUE TITLE
Test curl doesn't seem to work...

### ISSUE THREAD

**Issue by normalfaults** [ehazlett/conduit#3]
Deployed conduit with -r ehazlett/go-demo and token, passed port 8080 to host, tested with `curl -d '{"repository": {"repo_name": "ehazlett/go-demo"}}' http://localhost:8080?token=s3cr3+`

no response from conduit container 
conduit logs:

```
time="2015-05-15T00:23:02Z" level=info msg="starting conduit v0.0.4" 
time="2015-05-15T00:23:02Z" level=info msg="repos: [ehazlett/go-demo]" 
time="2015-05-15T00:23:02Z" level=info msg="tags: [latest]" 

```

go-demo container isn't deployed and isn't in docker ps history...

**Comment by ehazlett** [comment]
We need to update the sample curl.  It should be:

```
curl -d '{"repository": {"repo_name": "ehazlett/go-demo"}}' http://<host-machine-ip-or-docker-host-ip>:8080?token=s3cr3+
```

**Comment by ehazlett** [comment]
Fixed in 371692fbc
